packages/suite/src/components/firmware/FirmwareOffer.tsx

```diff
@@ -114,81 +100,64 @@ const FirmwareOffer = ({ device, customFirmware, targetFirmwareType }: Props) =>
+                                            id="TR_VERSION"
+                                            values={{ version: parsedChangelog.versionString }}
+                                        />
+                                        <TrezorLink
+                                            size="small"
+                                            variant="nostyle"
+                                            href={
+                                                parsedChangelog.notes
+                                                    ? parsedChangelog.notes
+                                                    : changelogUrl
+                                            }
+                                        >
+                                            <Button
+                                                variant="tertiary"
+                                                icon="EXTERNAL_LINK"
+                                                alignIcon="right"
                                             >
-                                                <Button
-                                                    variant="tertiary"
-                                                    icon="EXTERNAL_LINK"
-                                                    alignIcon="right"
-                                                >
-                                                    View all
-                                                </Button>
-                                            </TrezorLink>
-                                        </ChangelogHeading>
-                                        <ChangesUl>
-                                            {/* render individual changes for a given version */}
-                                            {parsedChangelog.changelog.map(
-                                                change =>
-                                                    // return only if change is not an empty array
-                                                    change && <li key={change}>{change}</li>,
-                                            )}
-                                        </ChangesUl>
-                                    </ChangelogGroup>
-                                </Changelog>
-                            }
-                            placement="top"
-                        >
-                            <Version new data-test="@firmware/offer-version/new">
-                                {formattedNextFirmwareType}
-                                {newVersion}
-                            </Version>
-                        </Tooltip>
-                    ) : (
-                        <Version new>
-                            {formattedNextFirmwareType}
-                            {newVersion}
-                        </Version>
-                    )}
-                </VersionWrapper>
+                                                View all
```

Comment:
Could be translated, right?

Reply:
The changelog is in English anyway, but yeah... `Translation` added in 05d94686197da26dd84461fc9a927226b60b5cc2.